source/cl/source/extension/source/intel_unified_shared_memory/usm-exports.cpp

```diff
@@ -531,20 +531,24 @@ cl_int MemFillImpl(cl_command_queue command_queue, void *dst_ptr,
       // Push Mux fill buffer operation
       auto mux_error =
           ExamineUSMAlloc(usm_alloc, device, return_event, mux_buffer);
-      OCL_CHECK(mux_error, return CL_OUT_OF_RESOURCES);
+      OCL_CHECK(mux_error, return_event->complete(CL_OUT_OF_RESOURCES);
```

Comment:
Other code here tries to handle the possibility that `return_event` is a null pointer, this probably needs to do the same. This all looks far too easy to get wrong, would be good to have a think about how we can refactor this to put all the `return_event->complete` calls in a single place, but if we can't find anything I'm happy with just plugging up all the places where we forgot.